ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -165,6 +165,46 @@ open class PillButtonBar: UIScrollView {
         return true
     }
 
+    @objc public func disableItem(_ item: PillButtonBarItem) {
+        guard let index = indexOfButtonWithItem(item) else {
+            return
+        }
+
+        buttons[index].isEnabled = false
+    }
+    
+    @objc public func disableItem(atIndex index: Int) {
+        if index < 0 || index >= buttons.count {
+            return
+        }
+
+        buttons[index].isEnabled = false
+    }
+    
+    @objc public func enableItem(_ item: PillButtonBarItem) {
+        guard let index = indexOfButtonWithItem(item) else {
+            return
+        }
+
+        let button = buttons[index]
+        if (!button.isEnabled)
+        {
+            button.isEnabled = true
+        }
+    }
+    
+    @objc public func enableItem(atIndex index: Int) {
+        if index < 0 || index >= buttons.count {
+            return
+        }
+
+        let button = buttons[index]
+        if (!button.isEnabled)
```

Comment:
added to avoid changing the state if already in that state. As it leads to change in appearance.
Reverted.

---
In reply to: [459652044](https://github.com/microsoft/fluentui-apple/pull/133#discussion_r459652044) [](ancestors = 459652044)